### ISSUE TITLE
Spaces and brackets in Japanese tags

### ISSUE THREAD

**Issue by 7nik** [stalkerg/anime-pictures#86]
Some source (e.g. VNDB and wikis) use character/artist names with space (Japanese or east), but pixiv and as result Danbooru skip spaces. So this
```
柾木　勝仁
柾木 勝仁
柾木勝仁
```
should be the same for the search system.

And similar for brackets:
```
浅葱眞夜（あさぎしんや）
浅葱眞夜(あさぎしんや)
```

**Comment by stalkerg** [comment]
Danburu did it because long time not supported space in tags at all (very stupid limitation in engine)

**Comment by MiTereKun** [comment]
да, этим все сайты на buru страдают. Как известно текста на иероглифах так же пишутся без пробелов

А у нас в ссылках вместо "@" юникодовое значение %40 вписывается. А ещё в русском переводе блок "о авторе" правильнее назвать "об авторе" (это маленькие исправления, которые как мне кажется, не заслуживают отдельной темы)
